func/upgrade.sh

```diff
@@ -632,7 +632,8 @@ upgrade_filemanager() {
 upgrade_roundcube(){
     if [ -n "$(echo "$WEBMAIL_SYSTEM" | grep -w 'roundcube')" ]; then
         if [ -d "/usr/share/roundcube" ]; then
-            echo "[ * ] Roundcube: Unable to update. Updates are managed by apt.";
+            echo "[ * ] Roundcube: Unable to update. Updates are managed over apt.";
+            echo "      To update to the last version of Roundcube please run the command migrate_roundcube.sh located in: /usr/local/hestia/install/upgrade/manual/"
```

Comment:
`To upgrade to the latest version of Roundcube directly from upstream, from please run the command migrate_roundcube.sh located in: /usr/local/hestia/install/upgrade/manual/` is clearer to the end user.